### PR TITLE
Generate candidate names based on the Marvel Cinematic Universe

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: app/data/helpers/generators.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
+  if(sex === 'male') {
+    return faker.helpers.randomize([
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:8]: You're missing:

- Bruce
- Wade
- Harold
- Peter
- James
- Taneleer
- Herman

at least...

File: app/data/helpers/generators.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      'Pietro',
+      'Jacques',
+      'Harry',
+      'Luke',
+      'Victor',
+      'Samuel'
+    ])
+  } else {
+    return faker.helpers.randomize([
+      'Janet',
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:24]: You're missing:

- Virgina
- Kara Lynn
- Hela

at least... :-)

File: app/data/helpers/generators.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      'Susan',
+      'Julia'
+    ])
+  }
+}
+
+exports.lastName = (sex) => {
+  if(sex === 'Male') {
+    return faker.helpers.randomize([
+      'Stark',
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:43]: You're missing:

- Hogan
- Octavius
- Quill
- Schultz
- Tivan

at least...!

File: app/data/helpers/generators.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      'Monica',
+      'Barbara',
+      'Bonita',
+      'Susan',
+      'Julia'
+    ])
+  }
+}
+
+exports.lastName = (sex) => {
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:40]: Does the last name need to be gender-specific?

**Review by simonwhatley — CHANGES_REQUESTED** [review]

File: app/data/helpers/generators.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:6]: Can you sort the list for ease on the eyes?

File: app/data/helpers/generators.js (hunk 4 above)
Comment (simonwhatley) [reply app/data/helpers/generators.js:40]: And can you sort the list for ease on the eyes?

**Comment by simonwhatley** [comment]
Looking at the data, I think there need to be quite a few more names to make the application list look more random. There are a lot of similar names.

File: app/data/helpers/generators.js (hunk 1 above)
Comment (adamsilver) [reply app/data/helpers/generators.js:8]: I need surnames for all the first names haha

File: app/data/helpers/generators.js (hunk 4 above)
Comment (adamsilver) [reply app/data/helpers/generators.js:40]: okay

File: app/data/helpers/generators.js (hunk 5 above)
Comment (adamsilver) [reply app/data/helpers/generators.js:6]: okay

File: app/data/helpers/generators.js (hunk 2 above)
Comment (simonwhatley) [reply app/data/helpers/generators.js:24]: There's also

- Sarah Jane

File: app/data/helpers/generators.js (hunk 3 above)
Comment (simonwhatley) [reply app/data/helpers/generators.js:43]: There's also

- Foster

File: app/data/helpers/generators.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    'Cleese',
+    'Danvers',
+    'Douglas',
+    'Hogan',
+    'Juárez',
+    'Maximoff',
+    'Morse',
+    'Octavius',
+    'Odinson',
+    'Pietro',
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:65]: You risk getting a "Pietro Pietro"...

File: app/data/helpers/generators.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+      'Samuel',
+      'Steve',
+      'Taneleer',
+      'Thor',
+      'Tony',
+      'Victor',
+      'Wade'
+    ])
+  } else {
+    return faker.helpers.randomize([
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:30]: No Sarah!

File: app/data/helpers/generators.js [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+      'Patricia',
+      'Susan',
+      'Virgina',
+      'Wanda'
+    ])
+  }
+}
+
+exports.lastName = () => {
+  return faker.helpers.randomize([
```
Comment (simonwhatley) [inline app/data/helpers/generators.js:51]: No Foster! Wow!

**Review by simonwhatley — CHANGES_REQUESTED** [review]

**Review by simonwhatley — APPROVED** [review]
Despite the continued omission of Sarah...

**Pull request merged**